src/nvim/testdir/test_command_count.vim

```diff
@@ -103,6 +103,8 @@ endfunc
 
 func Test_command_count_2()
   silent! %argd
+  cd
```

Comment:
Vim doesn't have this.
`:cd` changes to the home directory in Unix system.
Why?

Reply:
I will remove it.